tests/test_requests.py

```diff
@@ -460,10 +460,20 @@ def test_HTTP_200_OK_PUT(self, httpbin):
         r = requests.put(httpbin('put'))
         assert r.status_code == 200
 
-    def test_BASICAUTH_TUPLE_HTTP_200_OK_GET(self, httpbin):
-        auth = ('user', 'pass')
-        url = httpbin('basic-auth', 'user', 'pass')
+    @pytest.mark.parametrize(
+        'username, password', (
+            ('user', 'pass'),
+            (u'имя'.encode('utf-8'), u'пароль'.encode('utf-8')),
+        ))
+    def test_BASICAUTH_TUPLE_HTTP_200_OK_GET(self, httpbin, username, password, auth_str):
+        auth = (username, password)
+        url = httpbin('get')
+        
+        r = Request('GET', url, auth=auth)
```

Comment:
It looks that the problem on httpbin side. We request same url in py2 and py3, and we set same authorization header in both cases.

May be skip the test with unicode auth in py2?